<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input the number of cards:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="card_spin_box">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>43</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input the number of players:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="player_spin_box">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>43</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_game_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>70</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>801</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>801</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0" columnstretch="0" rowminimumheight="0" columnminimumwidth="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="restart_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="score_text_browser">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>500</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minutes_number">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="seconds_number">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>540</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gametime:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>sec</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>731</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
